src/components/File.jsx

```diff
@@ -143,6 +145,7 @@ class File extends Component {
     const { filename, extension } = splitFilename(attributes)
     return (
       <div className={classes}>
+        { attributes.mime === 'image/jpeg' && <Preview thumbnail={`${cozy.client._url}${attributes.links.small}`} /> }
```

Comment:
afaik yes. We discussed it while grooming, I asked about other image format like SVG and it was said that it won't be supported (so far)